phpbridge/server.php

```diff
@@ -10,26 +10,30 @@
 
 declare(strict_types=1);
 
+if (file_exists('../vendor/autoload.php')) {
+    require_once '../vendor/autoload.php';
```

Comment:
If the bridge was installed with composer it should be this, I think:
```suggestion
if (file_exists(__DIR__ . '/../../../../vendor/autoload.php')) {
    require_once __DIR__ . '/../../../../vendor/autoload.php';
```
`__DIR__` to make it relative to `server.php` instead of to the working directory, and a bunch of extra `..`s because it's nested pretty deep.
Is that right, or were you think of another use case? This package is up on Packagist, but installing from there is maybe not very normal.